temporian/implementation/numpy_cc/operators/sample.cc

```diff
@@ -0,0 +1,55 @@
+#include <cstdint>
+#include <iostream>
+#include <map>
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <string>
+#include <vector>
+
+namespace {
+namespace py = pybind11;
+
+typedef size_t Indice;

```

Comment:
same nit as above, `Index` instead of Indice

Reply:
Let use use "Idx" instead, and let's discuss this on our next meeting.